Chart.yaml

```diff
@@ -0,0 +1,6 @@
+name: g8s-fluentd-cloudwatch
+version: 0.1.0
+appVersion: 0.1.
+description: A Fluentd CloudWatch Helm chart for Kubernetes.
+sources:
+- https://github.com/giantswarm/g8s-fluentd-cloudwatch
```

Comment:
We tend to not use `appVersion` `description` and `sources` 
You can put the sources in the READMED instead

Reply:
ok I fix it